src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -16,19 +16,19 @@ def post(self, request, sentry_app):
         if sentry_app.is_internal:
             return Response({"detail": "Cannot publish internal integration"}, status=400)
 
-        # For now, just send an email that a request to publish has been amde
-        message = "User %s of organization %s wants to publish %s" % (
+        message = "User %s of organization %s wants to publish %s\n" % (
```

Comment:
Could use translations.

Reply:
Oh I suppose this is internal huh

Reply:
@EvanPurkhiser It's an email that goes to us, definitely needs to stay in English :)